### PR TITLE
Make API aggregation part of openshift_ca

### PR DESCRIPTION
**Pull request by smarterclayton**
Take wire_aggregator and move it into openshift_ca. Removes the need for an additional restart during master setup and keeps CA generation centralized.

Extracted from #7486

@deads2k

### CODE REVIEW

**Comment by smarterclayton** [comment]
@ewolinetz

File: roles/openshift_ca/tasks/main.yml [hunk 1]
```diff
@@ -85,6 +98,19 @@
...
 
+# This should NOT replace the CA due to --overwrite=false when a CA already exists.
+- name: Create the front-proxy CA if it does not already exist
+  command: >
+    {{ hostvars[openshift_ca_host]['first_master_client_binary'] }} adm ca create-signer-cert
+    --cert="{{ openshift_ca_config_dir }}/front-proxy-ca.crt"
+    --key="{{ openshift_ca_config_dir }}/front-proxy-ca.key"
+    --serial="{{ openshift_ca_config_dir }}/ca.serial.txt"
+    --expire-days={{ openshift_ca_cert_expire_days }}
+    --overwrite=false
```
Comment (ewolinetz) [inline roles/openshift_ca/tasks/main.yml]: does this interfere with redeploying certs? wouldn't we want to overwrite if we are forcing a cert redeploy?

**Comment by ewolinetz** [comment]
@mtnbikenc

File: roles/openshift_ca/tasks/main.yml (hunk 1 above)
Comment (smarterclayton) [reply roles/openshift_ca/tasks/main.yml]: Same logic as for master cert redeploy - I *think* a higher level playbook wipes the directory.

**Comment by smarterclayton** [comment]
It's still referenced from the old upgrade script - are we ready to delete the v3_7 upgrade?

**Comment by michaelgugino** [comment]
> It's still referenced from the old upgrade script - are we ready to delete the v3_7 upgrade?

Due to the wide-sweeping nature of the changes this cycle, I don't think we can effectively support upgrades from 3.7 to 3.9 on the 3.10 release branch.

**Comment by smarterclayton** [comment]
I'll go ahead and delete the v3_7 dir then

**Comment by smarterclayton** [comment]
Updated with removal of v3_7 upgrade

**Comment by michaelgugino** [comment]
> Updated with removal of v3_7 upgrade

@smarterclayton I think you may have forgotten to actually push up the changes.

**Comment by smarterclayton** [comment]
Updated with wire_aggregator removed

**Comment by smarterclayton** [comment]
This just removes the file now (I had it for reference in the static pods pr)

**Comment by smarterclayton** [comment]
I will fix the upgrade logic in #7723 to disable this

**Comment by smarterclayton** [comment]
```
[{u'cmd': u'/bin/oc adm certificate approve csr-mbfzn', u'returncode': 0, u'results': {}}, {u'cmd': u'/bin/oc adm certificate approve node-csr-zZ8nL63X4JQgLB_rd8IHmXgqNraJ9G6p4jeAqCfWn_c', u'returncode': 1, u'results': {}, u'stderr': u'Error from server (Conflict): Operation cannot be fulfilled on certificatesigningrequests.certificates.k8s.io "node-csr-zZ8nL63X4JQgLB_rd8IHmXgqNraJ9G6p4jeAqCfWn_c": the object has been modified; please apply your changes to the latest version and try again\n', u'stdout': u'No resources found\n'}]
```

@enj can you help me fix this upstream (not sure conflict should ever cause these commands to fail)

/retest

**Pull request merged**